### PR TITLE
Remove imports of meta

### PR DESCRIPTION
**Pull request by xster**
As suggested in https://github.com/flutter/flutter/pull/9059#discussion_r108556239

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

File: packages/flutter/lib/src/foundation/binding.dart [hunk 1]
```diff
@@ -7,7 +7,7 @@ import 'dart:convert' show JSON;
 import 'dart:developer' as developer;
 import 'dart:io' show exit;
 
-import 'package:meta/meta.dart';
+import 'package:flutter/foundation.dart';
```
Comment (Hixie) [inline packages/flutter/lib/src/foundation/binding.dart]: this is a recursive import (foundation library importing itself).

File: packages/flutter/lib/src/foundation/change_notifier.dart [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:meta/meta.dart';
+import 'package:flutter/foundation.dart';
```
Comment (Hixie) [inline packages/flutter/lib/src/foundation/change_notifier.dart]: ditto

File: packages/flutter/lib/src/foundation/binding.dart (hunk 1 above)
Comment (xster) [inline packages/flutter/lib/src/foundation/binding.dart]: Thanks for reviewing. I assume it's ok for foundation to import meta?

Comment (Hixie) [inline packages/flutter/lib/src/foundation/binding.dart]: Yup. (It also reexports it.)

**Pull request merged**